src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -1,4 +1,5 @@
 require 'common/common'
+require 'membrane'
```

Comment:
it's not only returning the hash, but also update of the bosh_cpi version. Bosh::OpenStackCloud::Cloud inherits Bosh::Cloud, which now does not have membrane requirement. I've got the errors in standard unit tests, for instance
```
134) Bosh::OpenStackCloud::Cloud should raise an Exception if OpenStack volume is not found
       Failure/Error: cloud = mock_cloud do |fog|
       NameError:
         uninitialized constant Bosh::OpenStackCloud::Cloud::Membrane
       # ./lib/cloud/openstack/cloud.rb:935:in `rescue in validate_options'
       # ./lib/cloud/openstack/cloud.rb:893:in `validate_options'
       # ./lib/cloud/openstack/cloud.rb:31:in `initialize'
       # ./spec/spec_helper.rb:116:in `new'
       # ./spec/spec_helper.rb:116:in `mock_cloud'
       # ./spec/unit/snapshot_disk_spec.rb:121:in `block (2 levels) in <top (required)>'
```